### ISSUE TITLE
Some question about Table.6

### ISSUE THREAD

**Issue by z7d1** [shijxcs/LIFT#13]
Thanks for your work. 

In Table.6, you report the result of `Zero-shot CLIP`. Is the CLIP used in this item fine-tuned on the long-tail dataset?

**Comment by shijxcs** [comment]
Zero-shot CLIP directly uses the pre-trained weights, and is **not** fine-tuned on the long-tail dataset.

**Comment by z7d1** [comment]
Thank you, Dr. Shi. 

I try to reproduce the accuracy 68.3% of Zero-shot CLIP reported in the paper.  And I got the Acc: 67.73% (Many: 0.6866, med: 0.6710, few: 0.6682) in test set. 

The checkpoint used is ViT-B/16 provided by openai/clip, and used `fivecrop/tencrop` in test time ensemble. I used different random seeds but the results are always different from the results reported in the paper. Can you give me some advice? I would appreciate it.

**Comment by shijxcs** [comment]
It seems that the random seeds should not affect the results, since both `zero-shot CLIP` and `fivecrop/tencrop` are deterministic processes. One potential influencing factor mightbe the image transformation method (`transforms.Resize(..., interpolation=InterpolationMode.BICUBIC)` or `transforms.Resize(..., interpolation=InterpolationMode.BILINEAR)`). Besides, the hardware may also affect the results, so perhaps you can try using another device.

**Comment by z7d1** [comment]
I had tried different seeds, as well as fivecrop / tencrop, but it still resulted in the same gap with the reported results. Perhaps, the cause lies in the device or other factors. Anyway, thank you for your reply!
